recipes/semantikon/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "semantikon" %}
+{% set version = "0.0.4" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 98b22a3732aed291a3dfdfb0842e05c27cc3299169313c073ef78faf5e2953f8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+    - versioneer
+  run:
+    - python >={{ python_min }}
+
+test:
+  imports:
+    - semantikon
+  requires:
+    - pip
+    - python {{ python_min }}
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/pyiron/semantikon
+  summary: 'semantikon - Ontological type system'
+  description: |
+    In the realm of the workflow management systems, there are well defined inputs and outputs for each node. semantikon is a Python package to give scientific context to node inputs and outputs by providing type hinting and interpreters.
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
```

```suggestion
  license_file: LICENSE
```
It is LICENSE not LICENCE.txt being packet, try with this :) @samwaseda